zipline/assets/asset_db_migrations.py

```diff
@@ -101,9 +101,38 @@ def _downgrade_v1_to_v0(op, version_info_table):
 
     write_version_info(version_info_table, 0)
 
+
+def _downgrade_v2_to_v1(op, version_info_table):
+    """
+    Downgrade assets db by removing the 'auto_close_date' column.
+    """
+    version_info_table.delete().execute()
+
+    # Drop indices before batch
+    # This is to prevent index collision when creating the temp table
+    op.drop_index('ix_equities_fuzzy_symbol')
```

Comment:
It looks like this is copied from the migration above. @jfkirk do we need to add this to every downgrade?  I'd be a bit surprised if that were the case.